app/helpers/heatmap_helper.rb

```diff
@@ -1,61 +1,25 @@
+# This is a class of static helper methods for generating data for the heatmap
+# visualization. See HeatmapHelperTest.
+# See selectedOptions in SamplesHeatmapView for client-side defaults, and
+# heatmap action in VisualizationsController.
 module HeatmapHelper
   DEFAULT_MAX_NUM_TAXONS = 30
+  # Zscore is best for heatmaps because it weighs the frequency against the background
+  DEFAULT_TAXON_SORT_PARAM = 'highest_nt_zscore'.freeze
```

Comment:
Where did you get this information?
Z-score does not differentiate much by itself.